BraintreeDropIn.podspec

```diff
@@ -10,7 +10,7 @@ Pod::Spec.new do |s|
                        Check out our development portal at https://developers.braintreepayments.com.
   DESC
   s.homepage         = "https://www.braintreepayments.com/how-braintree-works"
-  s.documentation_url = "https://developers.braintreepayments.com/ios/start/hello-client"
+  s.documentation_url = "https://developer.paypal.com/braintree/docs/start/hello-client/ios/v5"
```

Comment:
At some point it would probably make sense to update [all references](https://github.com/search?q=org%3Abraintree+developers.braintreepayments.com&type=code) to `developers.braintreepayments.com` in the various repos.

Reply:
Good idea - I created a card to update those.